src/components/mixins/Leaderboard.js

```diff
@@ -56,17 +52,30 @@ module.exports = {
     });
   },
 
-  processPage: function(page) {
-    return {
-      id: page.id,
-      name: page.name,
-      url: page.url,
-      isoCode: page.amount.currency.iso_code,
-      amount:  page.amount.cents,
-      totalMembers: page.team_member_uids.length,
-      imgSrc: page.image.large_image_url,
-      medImgSrc: page.image.medium_image_url
-    };
+  checkHasContentCallback: function(leaderboard) {
```

Comment:
I'm just wondering why this is broken out into a separate method. Is it for tesitng?